### DISCUSSION TITLE
Nextcloud AIO Containers won’t start

### DISCUSSION THREAD

**Discussion by psychedelicu** [nextcloud/all-in-one#4345] [Questions] [answered]
Hi,

I have nextcloud aio installed on unraid, from community apps.
But my master container won’t start my nextcloud containers.

Of course, i can select autostart on unraid, but then the redis container get an error, because it needs to be started from master container.

:signal-handler (1709895857) Received SIGTERM scheduling shutdown…
8:M 08 Mar 2024 13:04:17.361 # Redis is now ready to exit, bye bye…

![image](https://github.com/nextcloud/all-in-one/assets/35727400/a3286996-780a-48aa-adca-3440ad705f58)

**Comment by szaimen** [comment, marked as answer]
This will be handled in https://help.nextcloud.com/t/nextcloud-aio-containers-dont-start-containers/184694
